File: java/oc/src/main/java/oc/bad/CommandProcessor.java
package oc.bad;

public class CommandProcessor {
    public static void process(String command){
        System.out.println("Starting Your Command");
        switch (command){
            case "sing":
                System.out.println(command+"ing: We passed upon the stair");
                break;
            case "dance":
                System.out.println(command+"ing: Gangnam style");
                break;
            case "play":
                System.out.println(command+"ing: Gangnam style");
                break;
        }
        System.out.println("Done Your Command");
    }
}

File: python/bad/CommandProcessor.py
class CommandProcessor:
    @staticmethod
    def process(command):
        print("Starting Command " + command)
        if command=="sing":
            print(command+"ing: We passed upon the stair")
        elif command=="dance":
            print(command+"ing: Gangnam style")
        else:
            print(command+" not supported")
        print("Ending Command " + command)
